packages/design-picker/src/__tests__/integration.test.tsx

```diff
@@ -1,6 +1,47 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { isEnabled } from '@automattic/calypso-config';
```

Comment:
We have to import this module to be able to mock it, but eslint marks it as unused, hence I disabled it.